docs/users/input_ui.md

```diff
@@ -14,16 +14,15 @@ includes projects shared with the user by other Mergin users. Projects
 can be downloaded or updated (to or from) Mergin from this screen.
 
 ## GPS
-By default, the map is always centred around the GPS signal. This will
-limit to users to pan the map to the other parts of projects. Pressing
-and holding GPS icon, will turn off the auto-centre mode.
+By default, the map centred around the project extent. Pressing
+and holding GPS icon, will turn on the GPS auto-centre mode.
 
-GPS signal color on the map denotes the accuracy threshold set by user
+GPS signal colour on the map denotes the accuracy threshold set by user
 within the Input settings (See Settings section).
 
 ## Record
-This option will allow user to capture the position of the GPS and save it
-as point or line. Pressing the Record button for the first time will
+This option will allow user to capture the poison of the GPS and save it
```

Comment:
poison? :scream: :-D

I think your branch got messed up a bit (because you started off older commit on master) and that brought some conflicts (I fixed poison->position earlier but your changes seem to bring it back). Best to start branch always from the latest master when starting editing.